### PR TITLE
Use the same installer script for Linux and Windows

### PR DESCRIPTION
**Pull request by grejppi**

### CODE REVIEW

File: actions/installer-noninteractive.qs [hunk 1]
```diff
@@ -30,7 +30,26 @@ Controller.prototype.DynamicTelemetryPluginFormCallback = function() {
...
+
+        case "mingw": {
+            qtPath = "D:\\Qt";
+            break;
+        }
+
+        case "msvc": {
+            qtPath = "D:\\Qt";
+            break;
+        }
```
Comment (SecondFlight) [inline actions/installer-noninteractive.qs]: Is there any reason for this not to use case fallthrough? Example below:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch

Comment (SecondFlight) [reply actions/installer-noninteractive.qs]: I've tested this and it works as expected:
```javascript
let a = 'a';
switch(a) {
    case 'a':
    case 'b': {
        console.log('ab');
    }
}
```

Comment (grejppi) [reply actions/installer-noninteractive.qs]: I considered that but it just didn't look as aesthetically pleasing😌

Comment (SecondFlight) [reply actions/installer-noninteractive.qs]: 🤷‍♀

**Pull request merged**